site/app/Resources/views/user/notification_inbox.html.twig

```diff
@@ -0,0 +1,54 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}
+    Notification Inbox
+{% endblock %}
+
+{% block pagepath %}Home &raquo; Notification Inbox{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('notification_inbox') }}
+{% endblock %}
+
+{% block content %}
+    <h1>Your notifications ({{ mgilet_notification_unseen_count(app.user) }})</h1>
+    <div class="inbox-header row">
+        <div class="text-right">
+            {# TODO: Update the URL to Notification Settings #}
+            <a class="inbox-settings" href="#">Notification Settings</a>
+        </div>
+    </div>
+    {# Render all notifications #}
+    <div class="all-notifications list-group">
```

Comment:
- Remove the list-group here (use it on ul only)
- Add a "row" attribute to have the same padding as the previous div.